README.md

```diff
@@ -11,7 +11,8 @@ There's also an
 Chrome and probably Safari).
 
 #### Note:
-- `uglify-js` only supports ECMAScript 5 (ES5). 
+- `uglify-js` only supports ECMAScript 5 (ES5).
+- Support for `const` is present but incomplete, and may not be transformed properly.
```

Comment:
Please add:

`const` in `uglify-js@2.x` has function scope and as such behaves much like `var`, unlike `const` in ES2015 (ES6) which has block scope.